pkg/api/server/v1alpha2/log/s3.go

```diff
@@ -155,38 +157,39 @@ func (s3s *s3Stream) ReadFrom(r io.Reader) (int64, error) {
 		return 0, err
 	}
 
-	size := s3s.partSize + n
+	size := s3s.totalSize + n
```

Comment:
question: why was this renamed from partSize to totalSize?

Reply:
Nothing in particular, "totalSize" seemed more relevant name, and for Minio tenant the ContentLength was required in the API input params later in the method uploadPart, which is named partSized. Thought this would avoid confusion.

Reply:
Reverted back.